src/network/types.rs

```diff
@@ -57,6 +57,10 @@ pub struct Network {
     /// Network DNS servers for aardvark-dns.
     #[serde(rename = "network_dns_servers")]
     pub network_dns_servers: Option<Vec<IpAddr>>,
+
+    /// outbound addr for this network.
+    #[serde(rename = "outbound_addr")]
+    pub outbound_addr: Option<IpAddr>,
```

Comment:
That is not the way to do it, no new fields on the network. This should be set as regual option in the options map.